x-pack/plugins/observability_solution/entity_manager/server/lib/entities/install_entity_definition.ts

```diff
@@ -57,6 +59,11 @@ export async function installEntityDefinition({
 
   try {
     logger.debug(`Installing definition ${JSON.stringify(definition)}`);
+
+    if (!validateDefinitionCanCreateValidTransformIds(definition)) {
+      throw new EntityIdTooLong('ID is too long; the resulting transform ID will be invalid');
```

Comment:
Should we include feedback about the max length allowed? To avoid users doing trial and error.

Reply:
yeh we can do. i'll have to refactor the validation function to throw the error though. which i think is fine.